docs/getting-started.md

```diff
@@ -13,7 +13,11 @@ We'll be going over the following steps:
 5. [Pushing new changes to the component](#5-pushing-new-changes-to-the-component)
 6. [Adding storage to the component](#6-adding-storage-to-the-component)
 
-**Extra documentation:**
+**Command completion**
```

Comment:
I would just put `command completion` under `Extra documentation`. No need for an additional section, imo.

Reply:
Sounds good to me

Reply:
Done